src/main/java/model/Product.java

```diff
@@ -20,8 +28,9 @@
 
     public Product() { }
 
-    public Product(String name, String description, int inventory, byte[] image, Long shopID)
+    public Product(String name, String description, int inventory, byte[] image, Long shopID, String imageUrl)
```

Comment:
If you're using the imageUrl String, do we still need the image byte array?

Reply:
no we dont, forgot to remove it.